app/src/util/connectors.ts

```diff
@@ -88,7 +88,8 @@ class WalletConnectConnector extends Connectors.Connector {
 const WalletConnect = new WalletConnectConnector()
 
 const Infura = new NetworkOnlyConnector({
-  providerURL: infuraNetworkURL,
+  providerURL:
+    localStorage.getItem('relay') === 'true' ? networks[networkIds.XDAI].url : networks[networkIds.MAINNET].url,
```

Comment:
@hexyls Hey men I added this as well I was having some issues where incorrect url was being checked and modal wasn't showing up...and this fixed it. What do you think?

Reply:
looks good 👍